src/main/java/org/kohsuke/github/GHPullRequest.java

```diff
@@ -19,20 +19,14 @@
  */
 package org.kohsuke.github;
 
-import javax.annotation.CheckForNull;
+import javax.annotation.Nullable;
 import java.io.IOException;
 import java.net.URL;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Date;
-import java.util.List;
-
-import static org.kohsuke.github.Previews.*;
+import java.util.*;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Explicitly import the specific classes needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2208)